test/test_support.ts

```diff
@@ -34,26 +37,20 @@ const {cachedLibPath, cachedLib} = (function() {
   let host = ts.createCompilerHost(compilerOptions);
   let fn = host.getDefaultLibFileName(compilerOptions);
   let p = ts.getDefaultLibFilePath(compilerOptions);
-  return {cachedLibPath: p, cachedLib: host.getSourceFile(fn, ts.ScriptTarget.ES2015)};
+  return {
+    cachedLibPath: path.normalize(p),
+    cachedLib: host.getSourceFile(fn, ts.ScriptTarget.ES2015)
+  };
 })();
 
 /** Creates a ts.Program from a set of input files. */
 export function createProgram(sources: Map<string, string>): ts.Program {
   let host = ts.createCompilerHost(compilerOptions);
 
-  // Fake out host.directoryExists so that it doesn't read through node_modules/@types.
-  let realDirectoryExists = host.directoryExists;
-  host.directoryExists = dirName => {
-    if (path.isAbsolute(dirName)) {
-      let relName = path.relative(process.cwd(), dirName);
-      if (relName === 'node_modules/@types') return false;
-    }
-    return realDirectoryExists ? realDirectoryExists(dirName) : false;
-  };
-
   host.getSourceFile = function(
                            fileName: string, languageVersion: ts.ScriptTarget,
                            onError?: (msg: string) => void): ts.SourceFile {
+    fileName = path.normalize(fileName);
```

Comment:
Why are these normalize() calls necessary?  Is it worth explaining in a comment?  (I'm not sure, what do you think?)